editor/xml_feature.hpp

```diff
@@ -0,0 +1,91 @@
+#pragma once
+
+#include "geometry/point2d.hpp"
+
+#include "coding/multilang_utf8_string.hpp"
+
+#include "base/string_utils.hpp"
+
+#include "std/ctime.hpp"
```

Comment:
Why do you need iostream, map, ctime and point2d here? Please, cleanup your includes.